code/modules/client/verbs/who.dm

```diff
@@ -80,6 +80,8 @@
 		for(var/client/C in admins)
 			if(!C.holder.fakekey)
 				msg += "\t[C] is a [C.holder.rank]\n"
+			if(C.is_afk())
+				msg += " (AFK)"
```

Comment:
This needs to be tabbed one level, otherwise afk stealth mins would show an empty (AFK) line